bindings/zeebe/command/metadata.yaml

```diff
@@ -15,7 +15,9 @@ binding:
     - name: topology
       description: "Obtains the current topology of the cluster the gateway is part of."
     - name: deploy-process
-      description: "Deploys a single process to Zeebe."
+      description: "Deploys a single process to Zeebe. This is deprecated! Please use deploy-resource instead."
```

Comment:
```suggestion
      description: "Deprecated alias of 'deploy-resource'."
```

Reply:
@ItalyPaleAle What do you suggest regarding the documentation itself? Should we only document "deploy-resource" there or also the alias?

Reply:
The reason why I have changed the operation is that Camunda has deprecated the DeployProcess API: https://github.com/camunda/zeebe/blob/71d2e6accc9f0452c72c1de7c16f18cc24d22552/clients/go/pkg/commands/deploy.go#L53

Only adding an alias will not work, because the response for the operation is also different. Actually the API is still alpha. Would it be possible to add a breaking change here and remove the deploy-process alias completely? I don't think it make sense to use a deprecated API in a stable API later.

Reply:
Ok I see.

Perhaps, could it still be supported for now, but showing a warning that is deprecated and will be removed in the next Dapr version? (As a separate operation just like it does today).

Although we can deprecate things in alpha components, if we can avoid that (or do that gracefully) it's always best

Reply:
OK, I can do that

Reply:
I'm ok with dropping operations in going from Alpha to Stable. It's up to the two of you @ItalyPaleAle and @akkie. For now I will merge this PR and this can be addressed in a follow-up PR.

Reply:
@ItalyPaleAle Sorry, my bad. The actual deploy-process command in master uses already the deployResource functionality under the hood. So an alias would be completely sufficient. So lets keep the alias for the moment and then remove it later as @berndverst suggested.